lms/djangoapps/instructor_task/tasks.py

```diff
@@ -176,3 +177,14 @@ def get_ora2_responses(entry_id, xmodule_instance_args):
     action_name = ugettext_noop('generated')
     task_fn = partial(push_ora2_responses_to_s3, xmodule_instance_args)
     return run_main_task(entry_id, task_fn, action_name)
+
+
+# pylint: disable=not-callable
+@task(base=BaseInstructorTask, routing_key=settings.ORA2_RESPONSES_DOWNLOAD_ROUTING_KEY)
```

Comment:
I didn't want to create another celery queue and figured the responses would be done pretty quickly. Should I create a new one?